demo/predictions/js/path.js

```diff
@@ -0,0 +1,43 @@
+// From Bonita Day 2023 Demo
+class PathResolver {
```

Comment:
**issue:** we cannot duplicate the code from the demo.
We already know it requires refactoring at least for https://github.com/process-analytics/bonita-day-demo-2023/issues/52. There are also a lot of extra validation of the incoming/outgoing (verification that a related edge id exists in the model) whereas this should be guarantee by bpmn-visualization.
In addition, we plan to extract PathResolver from the bonita day demo https://github.com/process-analytics/bonita-day-demo-2023/issues/36. If we duplicate the implementation here, we must track it and plan to use the shared code here. And this may not possible out of the box depending on the way we package the shared code. In particular, I don't plan to provide an IIFE bundle for the shared lib.

Reply:
To replace in another PR. See #260